### ISSUE TITLE
🙈 As a http://assemble.io/ mobile user, I'm also reading the page when I scroll up! #UXD #Bug #FAIL

### ISSUE THREAD

**Issue by ldexterldesign** [assemble/assemble#802]
Hey,

Hope you're well

🙈 As a http://assemble.io/ mobile user, I'm also reading the page when I scroll up! #UXD #Bug #FAIL ~ https://twitter.com/ldexterldesign/status/684494825442406401

Hope this is useful

Regards

**Comment by assemblebot** [comment]
@ldexterldesign Thanks for the issue! If you're reporting a bug, please be sure to include:

- The version of `assemble` you are using.
- Your assemblefile.js (This can be in a gist)
- The commandline output. (Screenshot or gist is fine)
- What you expected to happen instead.


If this issue is about using assemble with grunt, please open an issue
on the [grunt-assemble](https://github.com/assemble/grunt-assemble/issues/new) repository.

**Comment by jonschlinkert** [comment]
wrong repo #fail! and already reported lots of times! #fail#fail? but thanks!

**Comment by ldexterldesign** [comment]
Hey @jonschlinkert, 

Thanks for reply

Apologies, I got the wrong repo' - I will assume https://github.com/assemble/assemble.io/issues/134 is the correct place. Surely, would have taken less time to simply drop a URL in your reply if you knew where to locate the issue, but judging from your tone I doubt you even care about user feedback and who uses your product.

TBH, for such a crippling bug, especially on mobile, it's unbelivble the aforementioned issue is closed and hasn't been fixed in a year?!

I used Twitter, and linked it, because it easilly allows me to upload video screencasts

I've been researching SSGs recently* and looking forward to contributing ( https://twitter.com/ldexterldesign/status/642227433278717952 ), but, based on my initial impression of support, I think I'll look elsewhere now

Regards

* -
- https://github.com/assemble/assemble/
- https://github.com/docpad/docpad
- https://github.com/hexojs/hexo
- https://github.com/jnordberg/wintersmith
- https://github.com/laktek/punch
- https://github.com/metalsmith/metalsmith

**Comment by jonschlinkert** [comment]
First, I don't at all react badly to feedback in general. I like feedback and ask for it often. But sometimes it's easy to tell when a person is disingenuous. For example, I know my mom loves me because she says things like, "I love you, Jon", and there is never a `#FAIL` hashtag afterwards. 

The second twitter link you just shared makes me think that perhaps I mistook your intentions, and if so I apologize.  But if you want to come across as eager to help, my humble suggestion is to try sounding like you want to help instead of sounding critical. 

Also, your personal website shows that you do work for what appears to be a competing, commercial, site builder. This contributed to my perception. (I do want to thank you for saving me the time of creating those links for those awesome projects created by our fellow open source friends. I'll add that to our readme)

> TBH, for such a crippling bug, especially on mobile, it's unbelivble the aforementioned issue is closed and hasn't been fixed in a year?!

If we want to just focus on the merits of your points. You're absolutely right. The docs suck and need to be updated. But, all things considered we've had close to 1,000,000 visits on the site in the past year, with an average of 6 minutes per visit, and we've only had a handful of issues about it. From a relative perspective, there are other issues that are more pressing (like things that prevent users from doing their jobs, rather than preventing them from learning more). Here are some other things to take into consideration:

1. assemble.io was the very first website I created
1. That was the first code project I ever worked on (I learned how to program while creating assemble, with a lot of help from @doowb)
1. That was the first documentation I did for a code project 
1. The site was build when assemble was a grunt plugin, and we just released the new API this past week.

I've said this elsewhere, but since assemble tends to appeal to designers (much more so than other static site generators), we get comparatively **far fewer** contributions than other large projects we maintain. That's totally fine, we both completely understand (especially given that when we created this I myself did not know how to program). We love this project and will continue to try to improve it, but it would be unbelievably awesome to wake up one day and see a pull request with someone fixing that bug, instead of being haggled for not having done it. 

Until we have the physical time and ability to fix it, it is what it is...
